plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/SpanAddingClassVisitorFactory.kt

```diff
@@ -61,12 +58,12 @@ abstract class SpanAddingClassVisitorFactory :
                 parameters = parameters.get()
             )
             ?: nextClassVisitor.also {
-                logger.warn(
+                SentryPlugin.logger.warn {
```

Comment:
isn't possible to get via ctor right? not a big fun of statics

Reply:
yeah it's not :( I can make it a top-level property, but not sure if it's any better. Also seen this current pattern in other plugins like Firebase-perf, so it's probably fine